contracts/drafts/issuance/Issuance.sol

```diff
@@ -164,6 +168,15 @@ contract Issuance is Ownable, StateMachine, ReentrancyGuard {
         _transition("FAILED");
     }
 
+    /**
+     * @dev Function to transfer all collected tokens to the wallet of the owner
+     */
+    function transferFunds() public onlyOwner {
+        require(currentState == "LIVE", "Cannot transfer funds now.");
+        currencyToken.transfer(wallet, amountRaised);
```

Comment:
What if instead of `transfer` we use `transferFrom` and let the owner decide where to send them at runtime?

We could remove line 68 as well then.